pkg/cluster/delete.go

```diff
@@ -456,11 +456,13 @@ func (m *manager) Delete(ctx context.Context) error {
 		}
 	}
 
-	// Don't fail the deletion because of hive
-	// This should change when/if we start using hive for cluster deletion
-	err = m.hiveDeleteResources(ctx)
-	if err != nil {
-		m.log.Info(err)
+	if m.adoptViaHive || m.installViaHive {
+		// Don't fail the deletion because of hive
+		// This should change when/if we start using hive for cluster deletion
+		err = m.hiveDeleteResources(ctx)
+		if err != nil {
+			m.log.Info(err)
+		}
 	}
```

Comment:
Does this mean could lead ClusterDeployments?

Reply:
It would log it within the RP server logs, but no customer could access these logs.